src/register/register-gnome/gnucash-item-edit.c

```diff
@@ -505,6 +505,9 @@ gnc_item_edit_new (GnucashSheet *sheet)
                            NULL);
     gtk_layout_put (GTK_LAYOUT(sheet), GTK_WIDGET(item_edit), 0, 0);
 
+    // Set the style context for this widget so it can be easily manipulated with css
+    gtk_style_context_add_class (gtk_widget_get_style_context (GTK_WIDGET(item_edit)), "GncRegisterItemEdit");
+
```

Comment:
To elaborate a bit on this, ideally we should create nodes/classes (I'm always mentioning both due to the functions that only first appear in gtk 3.20) for

- the register itself
- the register header
- the register body
- a register row (optionally providing even/odd differentiation via flags)
- the (row) cursor
- the cell cursor

Each of these can take css style properties such as color, font, padding, border color, border width,...
We don't have to define all of those ourselves though. We should implement in a way these are taken into account when a theme designer sets them.

And I also think this may be more work than we have time for before 2.8. Yet we should lay at least the groundwork now that we are redoing the register. And these considerations go for all of our custom widgets by the way, but most of them are just a combination of existing widgets, so they style automatically. The register and the SX calendar overview are two exceptions because they are based on GtkLayout so that makes us responsible for properly handling all styling. (Which makes me wonder again whether going the GtkTreeView route may still be the better solution in the long run...)